contracts/utils/Imports.sol

```diff
@@ -0,0 +1,6 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import { InterchainRouter } from '@axelar-network/axelar-gmp-sdk-solidity/contracts/utils/InterchainRouter.sol';
+import { InterchainRouterProxy } from '@axelar-network/axelar-gmp-sdk-solidity/contracts/utils/InterchainRouterProxy.sol';
```

Comment:
once gmp sdk is released, we can skip these imports?

Reply:
no, they are needed for tests (hardhat won't know they exists unless they are built by `hardhat compile` and you need to import them somewhere.